pkgs/by-name/go/godot_4/package.nix

```diff
@@ -34,13 +36,13 @@
   withTarget ? "editor",
   withTouch ? true,
   withUdev ? true,
+  withWayland ? true,
```

Comment:
```suggestion
  withWayland ? false,
```

Reply:
What? The purpose of the PR is changing this to "true"!

Reply:
> What? The purpose of the PR is changing this to "true"!

No. The purpose of PR is adding Wayland support. The default option is X11. And X11 users don't need wayland. (And vice-versa.)